apm-sniffer/apm-agent-core/pom.xml

```diff
@@ -53,10 +53,8 @@
         <shade.io.netty.target>${shade.package}.${shade.io.netty.source}</shade.io.netty.target>
         <shade.io.opencensus.source>io.opencensus</shade.io.opencensus.source>
         <shade.io.opencensus.target>${shade.package}.${shade.io.opencensus.source}</shade.io.opencensus.target>
-        <shade.org.apache.http.source>org.apache.http</shade.org.apache.http.source>
-        <shade.org.apache.http.target>${shade.package}.${shade.org.apache.http.source}</shade.org.apache.http.target>
         <shade.org.apache.commons.source>org.apache.commons</shade.org.apache.commons.source>
-        <shade.org.apache.commons.target>${shade.package}.${shade.org.apache.http.source}
+        <shade.org.apache.commons.target>${shade.package}.${shade.org.apache.commons.source}
```

Comment:
![image](https://user-images.githubusercontent.com/5441976/58452942-c5a58480-814b-11e9-9d90-cddee216749a.png)

Based on the current agent.jar, we even don't depend on Apache commons too. Please remove these two and shade.

Reply:
ok,before,I added a commons-lang3 dependency myself, i remove it now